scripts/build/config.js

```diff
@@ -142,7 +142,26 @@ function getFileOutput(bundle) {
   return bundle.output || path.basename(bundle.input);
 }
 
-module.exports = coreBundles.concat(parsers).map((bundle) => ({
-  ...bundle,
-  output: getFileOutput(bundle),
-}));
+module.exports = coreBundles
+  .concat(parsers)
+  .map((bundle) => {
+    const baseBundle = {
+      ...bundle,
+      output: getFileOutput(bundle),
+    };
+    if (baseBundle.target === "universal" && baseBundle.bundler !== "webpack") {
+      return [
+        {
+          ...baseBundle,
+          format: "umd",
+        },
+        {
+          ...baseBundle,
+          format: "esm",
+          output: baseBundle.output.replace(".js", ".module.js"),
```

Comment:
Should be `.mjs`.

FYI: I saw you post the reason in `babel` project, but do we have the same problem?

Reply:
@fisker 

> FYI: I saw you post the reason in babel project, but do we have the same problem?

Yes. It seems that webpack 4 resolve `*.js` as CommonJS when there is any `*.mjs`. This will break `*.js` written as ESM. So we need to use `.module.js` instead of `.mjs` for now.

Reply:
Sorry, I don't understand this, why we should care about webpack?

Reply:
I'm sorry about not enough explanation. This is not a problem about Prettier's bundler.

A package that bundles Prettier as ESM with webpack would face this problem. Bundling the code below may cause an error because webpack wrongly resolves ESM with `.js` extension which prettier depends on as CJS.

```js
import prettier from "prettier/standalone.mjs";
import parserGraphql from "prettier/parser-graphql.mjs";

prettier.format("query {     }", {
  parser: "graphql",
  plugins: [parserGraphql],
});
```

Reply:
> A package that bundles Prettier as ESM with webpack would face this problem. Bundling the code below may cause an error because webpack wrongly resolves ESM with .js extension which prettier depends on as CJS.

No, using `import` and `export` automatically enable ESM

Reply:
I don't understand problem with webpack here, despite the fact that I am his contributor

Reply:
@evilebottnawi Thank you for reviewing! I'm not an expert of webpack/ESM and maybe misunderstanding.

So [this comment](https://github.com/reactjs/react-transition-group/pull/77#issuecomment-453940955) is not true?
I referred these workarounds: https://github.com/preactjs/preact/pull/1425, https://github.com/formatjs/formatjs/issues/1395

Reply:
@evilebottnawi @fisker Sorry, I noticed two things.

1. Prettier's standalone bundle doesn't resolve any dependencies. So we don't need to care about the webpack's behavior.

2. `exports` field is breaking change. Should revert it.

Applied these things in 7b02e1d and 2762bbf